VMWare/vmware-esxi/tests/NETWORK_SOURCE.json

```diff
@@ -0,0 +1,34 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "vmware-esxi",
+        "dialect_uuid": "2b13307b-7439-4973-900a-2b58303cac90"
+      }
+    },
+    "message": "Connection from 1.2.3.4 port 33398"
+  },
+  "expected": {
+    "message": "Connection from 1.2.3.4 port 33398",
+    "observer": {
+      "vendor": "VMware",
+      "product": "ESXi"
+    },
+    "event": {
+      "kind": "event",
+      "category": [
+        "connection"
```

Comment:
Category `connection` doesn't exist. [network](https://www.elastic.co/guide/en/ecs/8.6/ecs-allowed-values-event-category.html#ecs-event-category-network) is more suitable.